addon_service/configured_addon/link/models.py

```diff
@@ -20,34 +22,40 @@ def is_supported_resource_type(resource_type: int):
 
 
 class ConfiguredLinkAddon(ConfiguredAddon):
-    target_id = models.CharField()
-    int_resource_type = models.BigIntegerField(validators=[is_supported_resource_type])
+    target_id = models.CharField(default=None, null=True, blank=True)
+    int_resource_type = models.BigIntegerField(
+        default=None, null=True, blank=True, validators=[is_supported_resource_type]
+    )
 
     @property
-    def resource_type(self) -> SupportedResourceTypes:
-        return SupportedResourceTypes(self.int_resource_type)
+    def resource_type(self) -> SupportedResourceTypes | None:
+        if not self.int_resource_type:
```

Comment:
```suggestion
        if self.int_resource_type:
```
I think there is a typo here